testcharms/charm-repo/quantal/mysql/icon.svg

```diff
@@ -0,0 +1,335 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Given we have some charms there already I've found adding an icon simpler. We could convert all these tests to use in memory charms, I'd prefer not to do that in this charm. Also, I just aded the real mysql icon.